evaluation/check_predict.py

```diff
@@ -0,0 +1,41 @@
+#shows how to predict using the model and how it works
```

Comment:
Please document script purpose and usage using `argparse` instead.

Reply:
Have added the description using argparse.